src/vm/amd64/calldescrworkeramd64.S

```diff
@@ -45,9 +45,8 @@ NESTED_ENTRY CallDescrWorkerInternal, _TEXT, CallDescrWorkerUnwindFrameChainHand
         push_nonvol_reg rbp
         mov     rbp, rsp
         push_nonvol_reg rbx
-        lea     rsp, [rsp - 8]          // ensure proper alignment of the rsp 
-        set_cfa_register rbp, (4*8)
-
+        alloc_stack 8                   // ensure proper alignment of the rsp 
```

Comment:
I am sorry, you are right. There used to be set_frame rbp, 0 before the lea, but now I can see that it was removed as part of the stack frame changes you did last week.